src/generic/FormSwitchGroup.jsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
```

Comment:
`FormSwitchGroup` component is changed which is already used in many places have you checked for ripples?

Reply:
Yes. I had updated it to match the Figma design batter.